ethereum/dex/trades.sql

```diff
@@ -655,6 +655,27 @@ WITH rows AS (
 
         UNION ALL
 
+        -- 1inch LP
+        SELECT
+            evt_block_time,
+            '1inch LP' AS project,
+            '1' AS version,
+            'DEX' AS category,
+            sender AS trader_a,
+            NULL::bytea AS trader_b,
+            result AS token_a_amount_raw,
+            amount AS token_b_amount_raw,
+            NULL::numeric AS usd_amount,
+            dstToken AS token_a_address,
+            srcToken AS token_b_address,
+            contract_address AS exchange_contract_address,
+            evt_tx_hash AS tx_hash,
+            NULL::integer[] AS trace_address,
+            evt_index
+        FROM onelp."MooniSwap_evt_Swapped"
```

Comment:
FYI the table name is Mooniswap, lower case s

Reply:
Thx, fixed!